cypress/support/commands.js

```diff
@@ -130,6 +130,20 @@ Cypress.Commands.add('dispatch', (...args) => {
 		.invoke(silent, ...['dispatch', ...args])
 })
 
+Cypress.Commands.add('findBy',
```

Comment:
Hehe, well I don't know exactly if this "reduce[s] complexity of cypress commands" :sweat_smile:

I find this code pretty hard to parse.

Reply:
Fair enough. :)
I'm happy to explain it if you're interested.

Do you also find the usage hard to parse?
```js
Cypress.Commands.add('deleteCollective', (name) => {
    cy.dispatch(GET_COLLECTIVES)
    cy.store('state.collectives.collectives')
        .findBy({ name })
        .dispatch(TRASH_COLLECTIVE)
...
```
I understand that all three of these (`store`, `findBy` and `dispatch`) are very abstract and generic. I also find them quite powerful.

I think some of the complexity and also power comes from the use of *"child actions"*. They take the output of the other action as input - but that's pretty implicit so harder to parse. But I think i got used to it very fast.

The last line could also be written as...
```js
    .then(collective => {
      cy.dispatch(TRASH_COLLECTIVE, collective)
    })
```

My impression was that one ends up with lot's of `.then`s, `cy.`s and a deep nesting pretty quickly.

Reply:
I'll merge this for now and continue exploring alternatives.

Reply:
Absolutely fine with merging, thanks! And the usage is super straight-forward. I'm not opposed to those abstract functions at all, just wanted to mention that they bring quite some complexity. Thanks a lot for the extra explanation. After going through them a few times I think I understood their logic :innocent: